scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
```

Comment:
No I don't have the urls for the backups, @danfruehauf would be the one to know where they were stored and how frequently the database was getting backed up.

> The next sql statement in this file is a table update to animal_release. I don't believe the soft-delete function would cover this case.

No it certainly wouldn't, but in this case we're just changing locations from null to the corresponding release/capture location, so the impact would be minor anyway.